modules/cats/package.scala

```diff
@@ -0,0 +1,41 @@
+/* -
```

Comment:
To remain consitent with other libs copyright notices should be added to all files with the AuthoHeader plugin or similar. If you think the COPYING file is good enough and better then we should change it elsewhere, not sure what the best way to proceed is. In any case we should mention changes to the process here https://github.com/47deg/47-process/blob/master/engineering.md#project-provisioning so all libs do it all the same way. We do plan on automating and enforcing all these as part of an sbt plugin where we have up to date versions, copyright headers enforcements etc.